django_celery_results/backends/database.py

```diff
@@ -194,6 +194,8 @@ def _forget(self, task_id):
 
     def cleanup(self):
         """Delete expired metadata."""
+        if not self.expires:
```

Comment:
i would love to see unit test for this

Reply:
This function is not covered in current unit tests, I didn't consider it the responsibility of this PR to add missing coverage. However, I can add a unit test.

Reply:
you can add a basic unit test for this added change or that function as you like

Reply:
it would be helpful if you can add a basic unit test so we can merge with confidence